ext/geos_c_impl/geometry.c

```diff
@@ -831,6 +831,32 @@ method_geometry_simplify_preserve_topology(VALUE self, VALUE tolerance)
   return result;
 }
 
+static VALUE
+method_geometry_simplify_polygon_hull(VALUE self,
+                                      VALUE vertex_fraction,
+                                      VALUE is_outer)
+{
+  VALUE result;
+  RGeo_GeometryData* self_data;
+  const GEOSGeometry* self_geom;
+  VALUE factory;
+
+  unsigned int is_outer_uint = RTEST(is_outer) ? 1 : 0;
+
+  result = Qnil;
+  self_data = RGEO_GEOMETRY_DATA_PTR(self);
+  self_geom = self_data->geom;
+  if (self_geom) {
+    factory = self_data->factory;
+    result = rgeo_wrap_geos_geometry(
+      factory,
+      GEOSPolygonHullSimplify(
+        self_geom, is_outer_uint, rb_num2dbl(vertex_fraction)),
+      Qnil);
+  }
+  return result;
+}
+
```

Comment:
Should we add method guards around this too? Not sure if having the undefined `GEOSPolygonHullSimplify` symbol will break compilation in older versions of not guarded or would only cause a segfault if it's called.

Reply:
@keithdoggett , you are right 👍 
It's safer to wrap the method with a guard.

I have updated the PR.